### DISCUSSION TITLE
Remove hard dependency on Examine

### DISCUSSION THREAD

**Discussion by seanrockster** [umbraco/Umbraco-CMS#16846] [Features and ideas]
First don't misunderstand, I'm not calling it rotten. Just saying, would it make sense if searching was provider based so that we could simply use a different search library or service if we want to, one that we already use and are familiar with, one that doesn't have the issues we see in azure?

**Comment by nul800sebastiaan** [comment]
This duplicates https://github.com/umbraco/Umbraco-CMS/discussions/12493 - please vote / continue the discussion there.

I believe we have an item coming on the roadmap around abstracting out search better, but not sure when we'll be able to get to it.

For now https://examinex.online/ is great for Azure.
